apps/test/unit/templates/instructions/CommentAreaTest.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {expect} from '../../../util/configuredChai';
+import {UnconnectedCommentArea as CommentArea} from '@cdo/apps/templates/instructions/CommentArea';
+
+const DEFAULT_PROPS = {
+  disabledMode: false,
+  onCommentChange: () => {},
+  comment: 'Good Work!',
+  placeholderText: 'Add your comment here'
+};
+
+describe('CommentArea', () => {
+  it('If displayed mode, textarea is display only', () => {
+    const wrapper = shallow(
+      <CommentArea {...DEFAULT_PROPS} disabledMode={true} />
+    );
+
+    const confirmTextArea = wrapper.find('textarea').at(0);
```

Comment:
Suggesting [`first()`](https://airbnb.io/enzyme/docs/api/ShallowWrapper/first.html) for readability.
```suggestion
    const confirmTextArea = wrapper.find('textarea').first();
```